package.json

```diff
@@ -80,5 +81,8 @@
     "ts-jest": "^29.0.0",
     "typedoc": "^0.23.10",
     "typescript": "^4.0.0"
+  },
+  "jestSonar": {
```

Comment:
If we add the `jest-sonar-reporter`to the reporters list, we can move this configuration item there too :yum:

Reply:
Again, tried that, Jest threw errors because the jest-sonar-reporter API wasn't compatible.